Python/4-partial.py

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Is that a PEP requirement to have an empty line in the beginning of the file if there are no comments and imports?